Products/CMFEditions/StandardModifiers.py

```diff
@@ -1116,7 +1119,46 @@ def afterRetrieveModifier(self, obj, repo_clone, preserve=()):
 class BlobProxy(object):
     pass
 
-class SkipBlobs:
+
+class BlobModifier:
```

Comment:
please inherit from object, otherwise its an old style class, this does not matter here but we should not start using old style